LICENSE.md

```diff
@@ -1,159 +1,334 @@
+   the Licensed Material is, connected with, or sponsored, endorsed, or granted
+   official status by, the Licensor or others designated to receive attribution
+   as provided in Section 3(a)(1)(A)(i).
 
-   6. __No endorsement.__ Nothing in this Public License constitutes or may be construed as permission to assert or imply that You are, or that Your use of the Licensed Material is, connected with, or sponsored, endorsed, or granted official status by, the Licensor or others designated to receive attribution as provided in Section 3(a)(1)(A)(i).
+b. _**Other rights.**_
 
-b. ___Other rights.___
+1. Moral rights, such as the right of integrity, are not licensed under this
+   Public License, nor are publicity, privacy, and/or other similar personality
+   rights; however, to the extent possible, the Licensor waives and/or agrees
+   not to assert any such rights held by the Licensor to the limited extent
+   necessary to allow You to exercise the Licensed Rights, but not otherwise.
 
-   1. Moral rights, such as the right of integrity, are not licensed under this Public License, nor are publicity, privacy, and/or other similar personality rights; however, to the extent possible, the Licensor waives and/or agrees not to assert any such rights held by the Licensor to the limited extent necessary to allow You to exercise the Licensed Rights, but not otherwise.
+1. Patent and trademark rights are not licensed under this Public License.
 
-   2. Patent and trademark rights are not licensed under this Public License.
+1. To the extent possible, the Licensor waives any right to collect royalties
+   from You for the exercise of the Licensed Rights, whether directly or through
+   a collecting society under any voluntary or waivable statutory or compulsory
+   licensing scheme. In all other cases the Licensor expressly reserves any
+   right to collect such royalties.
 
-   3. To the extent possible, the Licensor waives any right to collect royalties from You for the exercise of the Licensed Rights, whether directly or through a collecting society under any voluntary or waivable statutory or compulsory licensing scheme. In all other cases the Licensor expressly reserves any right to collect such royalties.
+### Section 3 – License Conditions
 
-### Section 3 – License Conditions.
+Your exercise of the Licensed Rights is expressly made subject to the following
+conditions.
 
-Your exercise of the Licensed Rights is expressly made subject to the following conditions.
+a. _**Attribution.**_
 
-a. ___Attribution.___
+1. If You Share the Licensed Material (including in modified form), You must:
 
-   1. If You Share the Licensed Material (including in modified form), You must:
+   A. retain the following if it is supplied by the Licensor with the Licensed
+   Material:
 
-       A. retain the following if it is supplied by the Licensor with the Licensed Material:
+   i. identification of the creator(s) of the Licensed Material and any others
```

Comment:
From what I can see they also weren't earlier. Markdown doesn't support roman numerals by default I think it really depends how/where it's viewed. On GitHub at least in comments you can:

```
1. a
  1. a
  1. a
  1. a
1. a
```

which will result in:

1. a
    1. a
    1. a
    1. a
1. a

or add css in license document with custom styling.

```markdown
<style type="text/css">
    ol ol { list-style-type: upper-alpha; }
    ol ol ol { list-style-type: lower-roman; }
</style>
```

But still we would need to use automatic ordering (each list point would be `1.` in raw text)

Reply:
> But still we would need to use automatic ordering (each list point would be `1.` in raw text)

This isn't true:

```
1. a
   1. a
   2. a
   3. a
2. a
```

Renders as:

1. a
   1. a
   2. a
   3. a
2. a

Still, there is no way that I know of of using the roman numbering in source.

Perhaps renaming `LICENSE.md` to just `LICENSE` would solve some of the problems, but it may also create new ones. Let's leave it as is, it looks reasonably good in source form.